app/javascript/controllers/text_animation_controller.js

```diff
@@ -0,0 +1,20 @@
+import { Controller } from "@hotwired/stimulus"
+let i = 0;
+let txt = 'Journey through tech without a sweat!';
+let speed = 40;
```

Comment:
@Kameronjan  only feedback is, we may want to slow the speed down a bit- but no biggie.

Reply:
Good idea, I'll do some more testing tmw when I do the styling for the booking page :D